Cargo.toml

```diff
@@ -34,12 +34,15 @@ categories = ["command-line-utilities", "development-tools"]
 
 [dependencies]
 aho-corasick = "1.0.2"
+ansi-to-tui = "3.1.0"
 async-dup = "1.2.4"
 backoff = { version = "0.4.0", default-features = false }
 camino = "1.1.4"
 clap = { version = "4.3.2", features = ["derive", "wrap_help", "env", "string"] }
 command-group = { version = "2.1.0", features = ["tokio", "with-tokio"] }
+crossterm = { version = "0.27.0", features = ["event-stream"] }
 enum-iterator = "1.4.1"
+gag = "1.0.0"
```

Comment:
I'm using `gag` to capture the `ghciwatch` process' `stderr` in a buffer so I can render it with the TUI.

I would prefer to specify a `Write`r at the `tracing-subscriber` level, instead of hackily capturing the output after it's emitted, but `tracing-human-layer` only supports emitting 
to `stdout` or `stderr` today.